dynacred/src/Data.ts

```diff
@@ -158,7 +158,7 @@ export class Data {
             signers: [adminSigner],
         });
 
-        const ocs = new OnChainSecretRPC(bc);
+        await (new OnChainSecretRPC(bc)).authorizeRoster();
```

Comment:
Got "1002 unexpected error: ByzCoinID already authorised" when using this path twice